pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -841,7 +843,8 @@ func TestDisabledUserAccount(t *testing.T) {
 	s := scheme.Scheme
 	err := apis.AddToScheme(s)
 	require.NoError(t, err)
-	config := configuration.LoadConfig()
+	cfg, err := configuration.LoadConfig(test.NewFakeClient(t))
+	config := cfg
```

Comment:
same here, plus you forgot to check the error:
```suggestion
	config, err := configuration.LoadConfig(test.NewFakeClient(t))
	require.NoError(t, err)
```

Reply:
fixed here: a7cfb415bcaa14a4fe90ea746eb7dfc59185eb65 and here: c5066c27bced708ae2f020d83113d041fbe39690